packages/service-clients/end-to-end-tests/azure-client/package.json

```diff
@@ -66,9 +66,11 @@
 		"@fluidframework/map": "workspace:~",
 		"@fluidframework/matrix": "workspace:~",
 		"@fluidframework/sequence": "workspace:~",
+		"@fluidframework/server-services-client": "^4.0.0",
```

Comment:
I can't find where this is used. Maybe I'm missing something, otherwise remove?

Reply:
Removed